target/linux/ramips/dts/mt7621_mikrotik_routerboard-m33g.dts

```diff
@@ -105,11 +109,17 @@
 		port@1 {
 			status = "okay";
 			label = "lan1";
+
+			nvmem-cells = <&macaddr_hard 1>;
+			nvmem-cell-names = "mac-address";
 		};
 
 		port@2 {
 			status = "okay";
 			label = "lan2";
+
+			nvmem-cells = <&macaddr_hard 1>;
```

Comment:
```suggestion
			nvmem-cells = <&macaddr_hard 2>;
```
Looks like rbm33g has 3 MAC addresses available for the 3 ethernet ports:
https://flytec.com.py/arquivos/imgs/imgs_produtos/41a24-29320-2.jpg / https://www.flytec.com.py/produto/29320

Reply:
Done.